src/network/axiosClient.js

```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+import { toast } from "react-toastify";
+
+const axiosClient = axios.create();
+
+axiosClient.defaults.baseURL = `${import.meta.env.VITE_BASE_URL}`;
+
+axiosClient.interceptors.response.use( response => {
+  
+  return response;
+}, (error) => {
```

Comment:
single argument shouldn't be wrapped in parenthesis
 ```suggestion
}, error => {
```